README.md

```diff
@@ -66,7 +66,7 @@ cardano-metadata-submitter 19309eb9c066253cede617dc635223ace320ae0bbdd5bd1968439
 | `logo`   | a PNG image file                                                      | `--logo \| -l`   |
 
 ```console
-cardano-metadata-submitter 19309eb9c066253cede617dc635223ace320ae0bbdd5bd1968439cd0 \
+cardano-metadata-submitter entry 19309eb9c066253cede617dc635223ace320ae0bbdd5bd1968439cd0 \
   --ticker "GIL" \
   --url "https://finalfantasy.fandom.com/wiki/Gil" \
   --unit "2,cents" \
```

Comment:
unit it gone i guess... ?

Reply:
> request includes again the unit parameter, this is gone, isn't it?

Wow great catch. I knew I had to be careful of this but completely forgot to check. Thanks